### DISCUSSION TITLE
Can you tell me, How you create this project?

### DISCUSSION THREAD

**Discussion by zergreen** [pystardust/ani-cli#538] [General] [answered]
## I just want to know how you create ?

- What algorithm you use
- Structure
- How to use API to web

If I try to create my online web video . 
like : netflix-cli, hotstar-cli, youtube-cli, and etc .

**Comment by port19x** [comment, marked as answer]
https://github.com/Blatzar/scraping-tutorial

**Comment by eatmynerds** [comment]
> Hi @zergreen,
> 
> 
> 
> This project is built using a modular structure with APIs for communication. It may use algorithms for streaming optimization and recommendations. API integration helps connect services like video hosting and authentication.
> 
> 
> 
> If you're looking to build a web video platform, consider video CMS solutions, CDNs, and adaptive streaming techniques. Let me know if you need more details!

i'm gonna touch you
